networkx/algorithms/bipartite/tests/test_matching.py

```diff
@@ -82,3 +84,13 @@ def test_to_vertex_cover(self):
         matching = maximum_matching(self.graph)
         vertex_cover = to_vertex_cover(self.graph, matching)
         self.check_vertex_cover(vertex_cover)
+
+
+def test_eppstein_matching():
+    # Special test
```

Comment:
Either we can use a plugin like [disable-docstring](https://github.com/kumar303/disable-docstring) on our local system.
Otherwise, adding a plugin to nosetest is quite easy. We can add the plugin during the setup.